.github/workflows/i386-alp.yml

```diff
@@ -0,0 +1,38 @@
+name: i386 alpine
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "6 0 * * *"
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: setup alpine
+        uses: jirutka/setup-alpine@v1
+        with:
+          arch: x86
+          packages: >
+            build-base
+            libaec-dev
+            libgit2-dev
+            cmake
+      - name: run script
+        run: |
```

Comment:
It would be better to follow the cmake-script.yml file as a template instead of reinventing the workflows every time! Makes it so much harder to maintain.